app/views/ships/index.html.erb

```diff
@@ -15,19 +22,19 @@
         <div class="card-description">
           <h2><%= ship.name %></h2>
           <%# <%= cl_image_tag @ship.photo.key, height: 300, width: 400, crop: :fill %>
-            <p><%= ship.category%>
-              ∙
-              <%= ship.location %></p>
-            <p class="card-price"><%= ship.price %>
-              CHF</p>
-            <%# <td><%= link_to 'Edit', edit_ship_path(garden)</td> %>
-              <%# <td><%= link_to 'Destroy', ship, method: :delete, data: { confirm: 'Are you sure?' }</td> %>
-              </div>
-            </div>
-          <% end %>
-        <% end %>
+          <p><%= ship.category%>
+            ∙
+            <%= ship.location %></p>
+          <p class="card-price"><%= ship.price %>
+            CHF</p>
+          <%# <td><%= link_to 'Edit', edit_ship_path(garden)</td> %>
```

Comment:
supprimer commentaire inutile ou commenter la raison pour laquelle ces liens sont commentés, exemple: todo: add edit and destroy link